addons/web_editor/static/src/js/editor/rte.summernote.js

```diff
@@ -1097,25 +1096,24 @@ var SummernoteManager = Class.extend(mixins.EventDispatcherMixin, ServicesMixin,
      * @param {jQuery} $editable
      * @returns {Promise}
      */
-    saveCroppedImages: function ($editable) {
-        var defs = _.map($editable.find('.o_cropped_img_to_save'), async croppedImg => {
-            croppedImg.classList.remove('o_cropped_img_to_save');
-
-            const croppedAttachmentSrc = await this._rpc({
-                route: '/web_editor/crop_attachment',
+    saveModifiedImages: function ($editable) {
+        var defs = _.map($editable.find('.o_modified_image_to_save'), async el => {
+            const isBackground = !el.matches('img');
+            el.classList.remove('o_modified_image_to_save');
+            const newAttachmentSrc = await this._rpc({
+                route: `/web_editor/modify_image/${el.dataset.originalId}`,
                 params: {
-                    res_model: croppedImg.dataset.resModel,
-                    res_id: parseInt(croppedImg.dataset.resId),
-                    name: croppedImg.dataset.originalName + '.crop',
-                    data: croppedImg.getAttribute('src').split(',')[1],
-                    mimetype: croppedImg.dataset.mimetype,
-                    original_id: parseInt(croppedImg.dataset.originalId),
+                    res_model: $editable.data('oe-model'),
+                    res_id: $editable.data('oe-id'),
```

Comment:
Hm. I was trying to prevent having to save res_model and res_id in the DOM, since they're info that already comes from the DOM. I thought the method was called with every editable element but apparently that's not the case.

Could I use `closest('[oe-model]')` on the images? Or would you rather I go back to passing that info through the DOM?

Reply:
Whatever solution. But if you go with closest, why not just go from
```javascript
    _.map($editable.find('.o_modified_image_to_save'), async el => {
```
to
```javascript
    _.map($editable, async editableEl => {
        const resModel = ...
        const resID = editableEl.dataset.oeResId;
        _.map($(editableEl).find('.o_modified_image_to_save'), async el => {
```
(bon codé à l'arrache dans github mais tu vois ce que je veux dire)

Reply:
Oh! So the method is actually called with all the editable elements inside of a single jQuery collection. I thought it was called many times with a one-element jQuery initially, and I didn't get what you meant when you said it was a multi-element object. This makes more sense, so I don't need closest at all.